src/master-playlist-controller.js

```diff
@@ -454,8 +454,7 @@ export class MasterPlaylistController extends videojs.EventTarget {
     this.subtitleSegmentLoader_.on('error', () => {
       videojs.log.warn('Problem encountered with the current subtitle track. Switching' +
                        ' back to default.');
-      this.subtitleSegmentLoader_.abort();
-      this.setupSubtitles();
+      this.handleSubtitleError_();
```

Comment:
error message is the same in both handlers, probably best to move the warn message into the callback and not use an anon func